packages/atomic/atomic.0.1/opam

```diff
@@ -0,0 +1,28 @@
+opam-version: "2.0"
+name: "atomic"
+version: "0.1"
+synopsis:
+  "Compatibility package for OCaml's Atomic module, from 4.12"
+author: "OCaml maintainers"
+maintainer: "simon.cruanes.2007@m4x.org"
+license: "LGPL-2.1"
+build: [
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.03" & < "4.12"}
```

Comment:
These versions of ocaml are not available in this repository. Can the dependency from opentelemetry be dropped altogether?